source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -2,6 +2,35 @@
 
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
+## Release 2021-03-24
+
+### Improvements
+
+#### User Interface (UI)
+ - Added a UI field for restricted domains under **Site Configuration > Posts** in the System Console. Expanded the logic that determines whether a post has a preview or not.
```

Comment:
@amyblais Can you tell me which feature PR this is driven by, please? (It's not yet on my radar.)

Reply:
@cwarnermm https://github.com/mattermost/mattermost-server/pull/16869